salvo/src/lib/api/docker_volume.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package salvo;
+
+// This message defines the properties for a given mount. It is used
+// to generate the dictionary specifying volumes for the Python Docker
+// SDK
+message VolumeProperties {
+  // Defines a list of properties governing the mount in the container
+  string bind = 1;
```

Comment:
This is probably lack of knowledge on my end, but the names "bind" and "mode" don't help me much to understand the fields. Can we include examples of what they may contain? Once I see that I may be able to suggest better names.